index.js

```diff
@@ -136,4 +160,5 @@ module.exports.version = VERSION;
 module.exports.location = LOCATION;
 module.exports.spawnClangFormat = spawnClangFormat;
 
-if (require.main === module) main();
+// if (require.main === module) main();
```

Comment:
I presume this is an accidental leftover? I'll clean that up when merging.

Reply:
Whoops, cleaned that up